extractor/src/main/java/org/schabi/newpipe/extractor/stream/StreamInfo.java

```diff
@@ -131,6 +132,8 @@ private static StreamInfo extractStreams(StreamInfo streamInfo, StreamExtractor
         /* Load and extract audio */
         try {
             streamInfo.setAudioStreams(extractor.getAudioStreams());
+        } catch (ContentNotSupportedException e) {
```

Comment:
Hmm… is this really the proper way to do this?

Reply:
I found this [stackoverflow answer](https://stackoverflow.com/a/20355868/12680950) and then did that, but if you know a cleaner way if there is, I'd be pleased to change it.

Reply:
I think this is the cleanest way to do this. :+1:
Or at least the best way I know :)